version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.11-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.12"
```

Comment:
You don't need to increment this here, it will get incremented automatically when you sbt release.